config/application.rb

```diff
@@ -11,10 +11,8 @@ class Application < Rails::Application
     # Initialize configuration defaults for originally generated Rails version.
     config.load_defaults 5.0
 
-    # Custom directories with classes and modules you want to be autoloadable.
-    # config.autoload_paths += %W(#{config.root}/extras)
-    config.autoload_paths += %W(#{config.root}/lib)
-    config.autoload_paths += Dir["#{config.root}/lib/**/"]
+    config.enable_dependency_loading = true
```

Comment:
I believe this is not thread-safe.  Can we do `eager_load_paths` on the line below instead?

Reply:
yes, done and rebased

Reply:
Okay, but now we don't need `enable_dependency_loading`

Reply:
removed